static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -850,6 +850,10 @@ $quote-mark: 35px;
             background-color: $garnett-neutral-5;
         }
 
+        .rich-link.tone-media--item {
```

Comment:
This should probably be in this file: https://github.com/guardian/frontend/blob/dae274df3741ce52c1bf5b38e3482d294b0685e8/static/src/stylesheets/module/onward-garnett/_rich-links-enhanced.scss